### PR TITLE
CLB: Firkraag, Cunning Instigator and support

### LINKED ISSUE #850: Firkraag, Cunning Instigator

### PR DESCRIPTION
**Pull request by Northmoc**
A good start...

What other situations should be considered?

I know Trove of Temptation is lurking... I wonder if there is some way to refactor that keyword into the new MustAttack Static.

Closes #850

### CODE REVIEW

**Comment by Northmoc** [comment]
@Agetian I see you implemented Trove of Temptation 5 years ago... any insight on this?

File: forge-game/src/main/java/forge/game/card/CardProperty.java [hunk 1]
```diff
@@ -1677,6 +1678,10 @@ else if (property.equals("blocked")) {
             if (band == null || !band.getAttackers().contains(card)) {
                 return false;
             }
+        } else if (property.equals("mustAttack")) {
+            if (!card.isGoaded() && StaticAbilityMustAttack.entitiesMustAttack(card).isEmpty()) {
```
Comment (tool4ever) [inline forge-game/src/main/java/forge/game/card/CardProperty.java:1682]: The Problem is it needs to look back
If the Attacker gets goaded after being declared it doesnt matter

**Comment by Agetian** [comment]
> @Agetian I see you implemented Trove of Temptation 5 years ago... any insight on this?

Hmm, not off the top of my head, sadly :/ It's a good idea to refactor it that way though, I agree.

**Comment by tool4ever** [comment]
I believe I fixed the Trove case for now
More tests need to be done later, also not sure yet if a Static can be done nicely

**Comment by Agetian** [comment]
:+1:

**Review by tool4ever — APPROVED** [review]

**Pull request merged**